Clinical-Genomics/cg
# Adds inter view links

This PR adds links for commonly used relationships between  views.

How to test:
0. install on clinical-db clinical-api-stage 
1. open clinical-api-stage 
1. in ApplicationVersion click Sample-link
1. in Analysis click Application-link
1. in MicrobialSample click Invoice-link
1. in Pool click Invoice-link
1. in Sample click Invoice-link
1. in Delivery click Sample-link
1. in FamilySample click Family-link
1. in FamilySample click Sample-link

Expected outcome:
The link should have opened the named view with the specific item listed
